database/migrations/2020_04_18_110359_create_srd_notes_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSrdNotesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('srd_notes', function (Blueprint $table) {
+            $table->unsignedSmallInteger('id');
```

Comment:
I'm assuming there is a reason we don't need a big int here?

Reply:
We'll never have that many notes (Unsigned Small Int = 65535) and we take the IDs directly from the SRD.